.travis.yml

```diff
@@ -36,4 +36,5 @@ jobs:
 
 after_success:
   # push coverage results to Coveralls and Codecov
-  - julia -e 'using Pkg; import TaylorIntegration; joinpath(dirname(pathof(TaylorIntegration)), ".."); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder()); Codecov.submit(Codecov.process_folder())'
+  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
+  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(process_folder())'
```

Comment:
Don't you need `import TaylorIntegration; joinpath(dirname(pathof(TaylorIntegration)), "..");` in each line, after `using Pkg`?

Reply:
Instructions in [Coverage.jl](https://github.com/JuliaCI/Coverage.jl#tracking-coverage-with-codecovio) say that it's not necessary. Actually, I was just reading that it may be enough to add `codecov: true` and `coveralls: true` to .travis.yml instead of these two lines.

Reply:
Fair enough; go ahead and make the changes to see :-)